### PR TITLE
TLC 389: QA issues in replatformed engineering guide 

### PR DESCRIPTION
**Pull request by eric-gade**
## Changes proposed in this pull request:
See the list of fixes in [TLC Issue 389](https://github.com/18F/TLC-crew/issues/389)

## security considerations

These are minor fixes and there are no know security considerations

### CODE REVIEW

File: content/engineering/languages-runtimes/ruby.md
Comment (igorkorenfeld) [inline content/engineering/languages-runtimes/ruby.md:24]: Poorly formatted link for Rubocop configuration (extra space after the "]" is likely breaking it)

File: _includes/categorylinks.html
Comment (igorkorenfeld) [inline _includes/categorylinks.html:8]: Getting a 404 for all of the links on the federalist preview. I think we need to add the `url` filter, as in `{{ link.url | url }}`

File: _includes/categorylinks.html
Comment (igorkorenfeld) [inline _includes/categorylinks.html:4]: I know we looked at last week, but with fresh eyes it does seem like a lot of space before the heading, and I'd love to cut it down a little (go from 40 to 16). I think we may need to create a new css rule though to avoid fighting with specificity with `usa-prose`. I am thinking we add a class like `.heading-md` to our custom css file something like

```
.heading-md {
  include @u-font('sans', 'md');
  margin-top: units(2);
}
```

**Review by igorkorenfeld — CHANGES_REQUESTED** [review]
Looking good! A few changes to make before merging. The redirect thing I mentioned is likely related to [TLC issue 420 ](https://github.com/18F/TLC-crew/issues/420)

File: content/engineering/our-approach/architecture-reviews/data-act-pilot.md
Comment (eric-gade) [reply content/engineering/our-approach/architecture-reviews/data-act-pilot.md:5]: Ah, good catch. Are there a lot of url renames like this? I'm not sure what's best. Should we keep these the same as the existing guides just for now, then come back to it post-migration?

File: assets/_common/styles/custom-styles.scss
Comment (eric-gade) [reply assets/_common/styles/custom-styles.scss]: Fixed in d8717cd

File: content/engineering/our-approach/architecture-reviews.md
Comment (eric-gade) [reply content/engineering/our-approach/architecture-reviews.md]: Fixed in d8717cd

File: content/engineering/languages-runtimes/ruby.md
Comment (eric-gade) [reply content/engineering/languages-runtimes/ruby.md]: Fixed in d8717cd

File: _includes/categorylinks.html
Comment (eric-gade) [reply _includes/categorylinks.html]: Fixed in d8717cd

File: _includes/categorylinks.html
Comment (eric-gade) [reply _includes/categorylinks.html]: I took a stab at it here: 23792fb
  
Let me know what you think. The specificity of `.usa-prose h2` was getting in the way, so I added a second rule for the margin-top

File: content/engineering/about/about.md
Comment (eric-gade) [reply content/engineering/about/about.md]: I think I got this right in https://github.com/18F/guides/pull/64/commits/e60637c2dd57704b25f418fdc43620dc860badc4

File: _includes/categorylinks.html
Comment (igorkorenfeld) [reply _includes/categorylinks.html]: Looks like this one is still broken – I think you put the url filter on the titles rather than on the `href` value of the `a` tag.

File: _includes/categorylinks.html
Comment (igorkorenfeld) [reply _includes/categorylinks.html]: Looking great!

File: content/engineering/our-approach/architecture-reviews/data-act-pilot.md
Comment (igorkorenfeld) [reply content/engineering/our-approach/architecture-reviews/data-act-pilot.md:5]: I am not sure what is the best path here. It has come up as part of a different [issue](https://github.com/18F/TLC-crew/issues/338), and I saw again that it's mentioned in [Anita's presentation ](https://docs.google.com/presentation/d/1c6W2zBvlUV8L-hgMqIAhXXmuhfzh5teP100rvHsur00/edit#slide=id.g275d934f5af_0_16), but I am don't know if we came to a decision. Let's check with Alex when he is back, because I think he may have a clearer picture from the different conversations.

File: content/engineering/about/about.md
Comment (igorkorenfeld) [reply content/engineering/about/about.md]: Thanks for organizing these, it's looking great! Only wondering now if `categorylinks.html` should is also just an engineering include and should be in that subfolder? I'll leave it up to you.

File: _includes/categorylinks.html
Comment (eric-gade) [reply _includes/categorylinks.html]: Yikes! Should be good now: https://github.com/18F/guides/pull/64/commits/b146a1b8f8f33eeaeb9179f0a71f4c369ee848e8 but will check the Pages build

File: content/engineering/about/about.md
Comment (eric-gade) [reply content/engineering/about/about.md]: Yes you are right, done in 9fb2096

File: _includes/categorylinks.html
Comment (igorkorenfeld) [reply _includes/categorylinks.html]: Looks good now!

File: content/engineering/about/about.md
Comment (igorkorenfeld) [reply content/engineering/about/about.md]: Thank you!

**Comment by adunkman** [comment]
Deferring my review to Igor or Amy — appreciate the work, and thanks y’all!

**Review by igorkorenfeld — APPROVED** [review]
The URL issue can be follow on work. It's mentioned in [TLC/338](https://github.com/18F/TLC-crew/issues/338), can be tied in with that work.

**Pull request merged**
